src/map/packets/message_system.h

```diff
@@ -26,6 +26,53 @@
 
 #include "basic.h"
 
+// Found in ROM/27/76.dat
```

Comment:
You could add a comment here saying the contents of this DAT are dumped into `MessageSystemIDs.log`